### PR TITLE
Update MLP ui-lib package version

### PR DESCRIPTION
**Pull request by terryyylim**
**What this PR does / why we need it**:

MLP repository was migrated in https://github.com/caraml-dev/mlp/pull/81, but the MLP app `ui-lib` dependency was not correctly updated.

**Which issue(s) this PR fixes**:

N/A

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: ui/yarn.lock [hunk 1]
```diff
@@ -1119,6 +1119,27 @@
   resolved "https://registry.yarnpkg.com/@bcoe/v8-coverage/-/v8-coverage-0.2.3.tgz#75a2e8b51cb758a7553d6804a5932d7aace75c39"
   integrity sha512-0hYQ8SB4Db5zvZB4axdMHGwEaQjkZzFjQiN9LVYvIFB2nSUHW9tYpxWriPrWDASIxiaXax83REcLxuSdnGPZtw==
 
+"@caraml-dev/ui-lib@^1.7.5-build.5-59f13e1":
```
Comment (krithika369) [inline ui/yarn.lock:1122]: Oh why is this added here?

File: ui/packages/app/package.json [hunk 2]
```diff
@@ -1,13 +1,13 @@
 {
   "name": "mlp-ui",
-  "version": "1.7.0",
+  "version": "1.7.6",
   "private": true,
   "license": "Apache-2.0",
   "dependencies": {
+    "@caraml-dev/ui-lib": "^1.7.5-build.5-59f13e1",
```
Comment (krithika369) [inline ui/packages/app/package.json:7]: I guess we can't use `@caraml-dev/ui-lib` @ `v1.7.6` until it's published. If so, should we update the MLP app UI in a separate PR after the lib is released?

Comment (terryyylim) [reply ui/packages/app/package.json:7]: If we don't point this to the dev build, the CI will fail because there's no such version `v1.7.0`.

Comment (krithika369) [reply ui/packages/app/package.json:7]: Ok.. I assume that after we release `v1.7.6`, we will update the MLP UI app deps again? If so, then LGTM.

**Review by krithika369 — APPROVED** [review]

File: ui/packages/app/package.json (hunk 2 above)
Comment (terryyylim) [reply ui/packages/app/package.json:7]: Yes that's correct!
